test/Dialect/HW/basic.mlir

```diff
@@ -227,3 +227,9 @@ module {
     hw.instance "h2" sym @inst_0 @C() -> () {circt.globalRef = [#hw.globalNameRef<@glbl_D_M1>, #hw.globalNameRef<@glbl_D_M2>, #hw.globalNameRef<@glbl_D_M3>]}
   }
 }
+
+module {
+  hw.testmodule @NewStyle (input %a : i3, output %b : i3, inout %c : i64 {hw.exportPort = #hw<innerSym@symA>}) {
```

Comment:
Is it worth parsing/printing `output` ports without the `%` since they're not actually ssa values?

Reply:
There's a couple other bits unsolved here related to ports with invalid SSA names.  I've been pushing that a bit down the road.